tools/unitctl/README.md

```diff
@@ -77,21 +115,32 @@ Note: Container will be on host network
 
 ```
 
-To the subcommand `unitctl instances new` the user must provide three things:
-1. **A directory such as `/tmp/2`.**
-   The Unit container will mount this to `/var/run` internally.
-   Thus, the control socket and pid file will be accessible from the host.
-2. **A path to an application.**
+To the subcommand `unitctl instances new` the user must provide three arguments:
+1. **A means of showing the control API:**
+   There are two possibilities for this argument.
+   A filepath on which to open a unix socket,
+   or a TCP address.
+   - If a directory is specified the Unit container
+     will mount this to `/var/run` internally.
+     Thus, the control socket and pid file will be
+     accessible from the host. For example: `/tmp/2`.
+   - If a TCP endpoint is specified Unit will be configured
+     to offer its control API on the given port and address.
+     For example: `127.0.0.1:7171`.
+2. **A path to an application:**
    In the example, `$(pwd)` is provided. The Unit container will mount
    this READ ONLY to `/www/`. This will allow the user to configure
```

Comment:
Is there a way to provide that folder in RW mode rather than read only?

Reply:
not yet, pending https://github.com/nginx/unit/issues/1247